geode-docs/reference/topics/gemfire_properties.html.md.erb

```diff
@@ -753,6 +760,12 @@ See <a href="../../developing/distributed_regions/how_region_versioning_works.ht
 <td>S</td>
 <td><em>not set</em></td>
 </tr>
+<tr class="even">
+<td>validate-serializable-objects</td>
+<td>A boolean that defaults to false. When true, all internal <%=vars.product_name%> classes and the objects defined by the property serializable-object-filter will be serialized. An <code>IncompatibleClassException</code> is thrown for objects not listed. JDK 8 build 121 or a later build must be installed to use this property. Servers and clients that do not meet this requirement will throw an exception upon startup.</td>
```

Comment:
I think this should say "instances of classes that are not internal <%=vars.product_name%> and whose class name is not allowed by the serializable-object-filter will not be permitted to be deserialized."

The property controls what is allowed to be deserialized, not what is allowed to be serialized.